packages/ra-ui-materialui/src/input/AutocompleteInput.spec.tsx

```diff
@@ -439,34 +474,6 @@ describe('<AutocompleteInput />', () => {
         expect(setFilter).toHaveBeenCalledWith('');
     });
 
-    it('should allow customized rendering of suggesting item', () => {
```

Comment:
Not sure why you would need to remove this test. To me it should still work fine.

Reply:
You were very right.